build.sbt

```diff
@@ -1,22 +1,33 @@
-import play.sbt.PlayScala
 import sbt.Keys._
-import sbt._
+import uk.gov.hmrc.SbtArtifactory
+import uk.gov.hmrc.playcrosscompilation.PlayVersion.{Play25, Play26}
 import uk.gov.hmrc.versioning.SbtGitVersioning
 import uk.gov.hmrc.versioning.SbtGitVersioning.autoImport.majorVersion
-import uk.gov.hmrc.SbtArtifactory
 
 
 val appName = "play-language"
 
 lazy val playLanguage = (project in file("."))
   .enablePlugins(PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtArtifactory)
+  .disablePlugins(PlayLayoutPlugin)
   .settings(
     majorVersion := 4,
     name := appName,
     scalaVersion := "2.11.12",
-    crossScalaVersions := Seq("2.11.12"),
+    crossScalaVersions := Seq("2.11.12", "2.12.10"),
     libraryDependencies ++= AppDependencies.all,
     resolvers += Resolver.typesafeRepo("releases"),
     resolvers += Resolver.bintrayRepo("hmrc", "releases"),
-    makePublicallyAvailableOnBintray := true
+    makePublicallyAvailableOnBintray := true,
+    PlayKeys.playPlugin := true
```

Comment:
Can you explain what this gives you?

Reply:
It's actually no longer required so I'll remove. Thanks for pointing out.